### PR TITLE
Create account screen

### PR DESCRIPTION
**Pull request by henryhu1**
just the screen

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt [hunk 1]
```diff
@@ -1,13 +1,479 @@
...
+					) {
+						Text(
+							text = stringResource(id = uiState.gender.stringId),
+						)
+
+						Spacer(modifier = Modifier.weight(1.0f))
+
+						Icon(
+							imageVector = Icons.Filled.KeyboardArrowDown,
+							contentDescription = "genders"
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt]: This string should be a resource

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt [hunk 2]
```diff
@@ -1,13 +1,479 @@
...
+					.height(50.dp),
+				onClick = {},
+				colors = ButtonDefaults.buttonColors(
+					containerColor = subletrPink,
+					contentColor = Color.White,
+				)
+			) {
+				Text(
+					text = stringResource(id = R.string.create_account),
+					color = Color.White,
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt]: Can you use the defined constant for this?
I think it's called `textOnSubletrPink` or something like that.
It'll just make our lives easier if we want to implement dark mode

**Review by leeminkyu16 — APPROVED** [review]
Just some small changes (sorry, I guess I keep missing things ;-;)
Otherwise, Let's Gamble, Try Merging

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt (hunk 2 above)
Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt]: I just added that constant for white text color and am using it now

Comment (henryhu1) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt]: disregard that i need to resolve conflicts lol

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

**Review by leeminkyu16 — APPROVED** [review]
I think there are definitely things here that we'll need to change to reflect some changes that we've made overall (using `R.dimen` and stuff like that) but I think you've suffered enough on this page for now.

Let's merge this for now and then fix the small issues later.

Let's Gamble, Try Merging.

**Pull request merged**